GoogleMapsApi/StaticMaps/StaticMapsEngine.cs

```diff
@@ -28,13 +28,13 @@ public string GenerateStaticMapURL(StaticMapRequest request)
 
 			var parametersList = new QueryStringParametersList();
 
-            if (!string.IsNullOrEmpty(request.ApiKey))
```

Comment:
Was this intentional? If not, please revert.

Reply:
Visual Studio automatically formatting it.
Could you tell me how to revert?